workflow/rules/population.smk

```diff
@@ -0,0 +1,44 @@
+if is_activated("population/db"):
```

Comment:
Does it make any difference to restrict these rules being added to the workflow only if a population-db is activated?